src/pages/public/HelperRegister.tsx

```diff
@@ -0,0 +1,358 @@
+  },
+  {
+    title: "Fertig!",
+    Icon: CheckCircle,
+    okText: "Weiter"
+  }
+];
+
+const contents: (StepContent | StepContent[])[] = [
+  {
+    stepIndex: 0,
+    Content: () => (
+      <>
+        <p className="mb-8">
+          Bevor du fortfährst: Zuverlässigkeit steht an oberster Stelle! Wenn du
+          kurzfristig keine Zeit hast, sag bitte direkt der Kontaktperson ab,
+          damit sie umplanen kann. Weiterhin musst du:
+        </p>
+        <ul className="mb-4">
+          {checklistTexts.map((text, i) => (
+            <li key={i} className="flex align-top pl-8 mb-2">
+              <div className="inline-block -ml-8 mr-2">
+                <CheckIcon />
+              </div>
+              <p className="inline-block font-medium">{text}</p>
+            </li>
+          ))}
+        </ul>
+      </>
+    )
+  },
+
+  {
+    stepIndex: 1,
+    Content: ({ setFieldValue, values }: PassedFormProps) => (
+      <>
+        {[
+          ["account.firstName", "Vorname"],
+          ["account.lastName", "Nachname"],
+          ["account.phone", "Telefonnummer"],
+          ["account.email", "E-Mail"],
+          ["account.password", "Passwort"],
+          ["account.password_confirmation", "Passwort wiederholen"]
+        ].map(([key, label]) => (
+          <TextField
+            key={key}
+            label={label}
+            className="mb-4 text-brand"
+            fullWidth
+            onChange={(e) => setFieldValue(key, e.currentTarget.value)}
+          />
+        ))}
+        <div className="mb-2">
+          <FormTitle as="h2" className="mb-2">
+            Du bist
+          </FormTitle>
+          <div>
+            {[
+              ["Schüler*in", "Schüler*in"],
+              ["Student*in", "Student*in"],
+              ["Arbeitslos", "Arbeitslos"],
+              ["Beschäftigt", "Beschäftigt"]
+            ].map(([label, value], i) => {
+              const checked = value === values.employmentStatus;
+
+              return (
+                <Radio
+                  key={i}
+                  checked={checked}
+                  onClick={() => setFieldValue("employmentStatus", value)}
+                  block
+                >
+                  {label}
+                </Radio>
+              );
+            })}
+          </div>
+        </div>
+      </>
+    )
+  },
+  {
+    stepIndex: 2,
+    Content: ({ setFieldValue, values }: PassedFormProps) => (
+      <>
+        <div className="mb-2">
+          <FormTitle as="h2" className="mb-2">
+            Du suchst etwas für
+          </FormTitle>
+          <div>
+            <Checkbox
+              className="mb-2"
+              checked={values.fullTime === true}
+              onChange={() => setFieldValue("fullTime", true)}
+              block
+            >
+              Vollzeit
+            </Checkbox>
+            <Checkbox
+              checked={values.partTime === true}
+              onChange={() => setFieldValue("partTime", false)}
```

Comment:
```suggestion
              onChange={() => setFieldValue("partTime", !values.partTime)}
```

Dann funktioniert auch die Checkbox.

Reply:
Ah, übersehen, danke dir!